src/main/webapp/app/exam/manage/test-runs/test-run-management.component.html

```diff
@@ -19,7 +19,7 @@ <h2><span jhiTranslate="artemisApp.examManagement.testRun.title">Test Run Manage
                     </button>
                 </div>
 
-                <div *ngIf="!exam.testExam" [ngbTooltip]="'artemisApp.examManagement.testRun.assessTestRunDisabled' | artemisTranslate" [disableTooltip]="testRunCanBeAssessed">
+                <div *ngIf="!exam?.testExam" [ngbTooltip]="'artemisApp.examManagement.testRun.assessTestRunDisabled' | artemisTranslate" [disableTooltip]="testRunCanBeAssessed">
```

Comment:
```suggestion
                <div *ngIf="exam && !exam.testExam" [ngbTooltip]="'artemisApp.examManagement.testRun.assessTestRunDisabled' | artemisTranslate" [disableTooltip]="testRunCanBeAssessed">
```
Otherwise this would also be true if there is no exam